src/main/external-resources/renderers/Sony-BraviaXBR.conf

```diff
@@ -24,6 +24,10 @@ UpnpDetailsSearch = Sony , XBR-[0-9][0-9]
 SeekByTime = true
 DLNALocalizationRequired = true
 TranscodeAudio = WAV
+#MaxVideoWidth = 3840
```

Comment:
Because i get bored to repeat myself, as more and more users with new renderers ask for that on the forum.
So now i just need to ask them to uncomment these lines in their renderer configuration file.
Anyway, they are just the reality of their renderer, so better add them, or someone will need to read their renderer specification again and again...

If you remark is about the comment sign, you're the reason. As i understood that you don't want set it by default. No problem, i can understand.